frontend/src/app/profile/profile-page/profile-page.component.html

```diff
@@ -0,0 +1,77 @@
+<!-- HTML Structure of Profile Page -->
+
+<div class="container">
+  <div class="profile-grid">
+    <!-- Profile Information Card -->
+    <div class="left-column">
+      <div class="profile-card-container">
+        <profile-about-card [profile]="profile" />
+      </div>
+      <mat-card *ngIf="profile.id" appearance="outlined">
+        <div *ngIf="profile.github !== ''; else associate_github">
+          <mat-card-header>
+            <img mat-card-avatar [src]="profile.github_avatar" />
+            <mat-card-title
+              >GitHub /
```

Comment:
In addition, since the GitHub profile icon is used on the top card, we can remove the image here!

The card could just be something like:

( Github Icon) GitHub: @username.   [Unlink]

*Note: See Rohan's code for adding the GitHub icon to the sidebar for adding this GitHub icon!)